docs/blog/rendered/ibis-version-6.1.0-release.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    t = t.mutate(t["year"].cast("str").name("year"))
</pre><p>Can you use the shorter syntax for naming new columns?</p><p> </p><pre class="ql-syntax" spellcheck="false">t = t.mutate(year=t["year"].cast("str"))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>